bagel/cli.py

```diff
@@ -748,3 +748,97 @@ def derivatives(
         },
         filename=output,
     )
+
+
+@bagel.command()
+def pheno_tsv(
+    pheno: Path = typer.Option(  # TODO: Rename argument to something clearer, like --tabular.
+        ...,
+        "--pheno",
+        "-t",  # for tabular
+        help="Path to a phenotypic .tsv file",
+        exists=True,
+        file_okay=True,
+        dir_okay=False,
+        resolve_path=True,
+    ),
+    dictionary: Path = typer.Option(
+        ...,
+        "--dictionary",
+        "-d",
+        help="Path to the .json data dictionary corresponding to the phenotypic .tsv file.",
+        exists=True,
+        file_okay=True,
+        dir_okay=False,
+        resolve_path=True,
+    ),
+    output: Path = typer.Option(
+        "pheno.tsv",
+        "--output",
+        "-o",
+        help="Path to save the output harmonized .tsv file.",
+        file_okay=True,
+        dir_okay=False,
+        resolve_path=True,
+    ),
+    overwrite: bool = overwrite_option(),
+    verbosity: VerbosityLevel = verbosity_option(),
+    help_: bool = help_option(),
+):
+    """
+    [red]Experimental command, please avoid using in production unless explicitly advised![/red]
+    Harmonize the contents of a tabular phenotypic file (.tsv) using the annotations defined in its corresponding Neurobagel data dictionary (.json, generated using the Neurobagel annotation tool).
+    """
+    collection_available_term = "nb:available"
+    collection_unavailable_term = "nb:unavailable"
+
+    file_utils.check_overwrite(output, overwrite)
+
+    data_dictionary = file_utils.load_json(dictionary)
+    pheno_df = file_utils.load_tabular(pheno)
+
+    logger.info("Running initial checks of inputs...")
+    # NOTE: `width` determines the amount of padding (in num. characters) before the file paths in the print statement.
+    # It is calculated as = length of the longer string + 2 extra spaces
+    width = 26
+    logger.info("%-*s%s", width, "Tabular file (.tsv):", pheno)
+    logger.info("%-*s%s", width, "Data dictionary (.json):", dictionary)
+    pheno_utils.validate_inputs(data_dictionary, pheno_df)
+
+    # TODO: Remove once we no longer support annotation tool v1 data dictionaries
+    data_dictionary = pheno_utils.convert_transformation_to_format(
+        data_dictionary
+    )
+
+    logger.info("Harmonizing raw tabular file...")
+
+    column_mapping = pheno_utils.map_categories_to_columns(data_dictionary)
+    collection_mapping = pheno_utils.map_tools_to_columns(data_dictionary)
+
+    output_columns = []
+    not_collection_output_columns = []
+    for std_var, columns in column_mapping.items():
```

Comment:
I feel like the variables could be named clearer here. 

`output_columns` -> `collections_columns`?
`not_collection_output_columns` -> `not_collection_columns`?
Or something like that. Otherwise I might be missing something here